app-games/src/prod/google-services.json

```diff
@@ -0,0 +1,29 @@
+{
+  "project_info": {
+    "project_number": "808759492170",
+    "project_id": "aptoide-games",
+    "storage_bucket": "aptoide-games.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:808759492170:android:4f9abf37305c970450f0b3",
+        "android_client_info": {
+          "package_name": "com.aptoide.android.aptoidegames.prod"
```

Comment:
Although this file will be replaced when the app games prod app is added to the firebase project, we should make sure the package name does not end in '.prod'. The package name in prod should be only `com.aptoide.android.aptoidegames`